Sources/SnapshotTesting/Snapshotting/URLRequest.swift

```diff
@@ -16,4 +16,44 @@ extension Snapshotting where Value == URLRequest, Format == String {
 
     return ([method] + headers + body).joined(separator: "\n")
   }
+  
+  /// A snapshot strategy for comparing requests based on a cURL representation.
+  public static let curl = SimplySnapshotting.lines.pullback { (request: URLRequest) in
+
+    var components = ["curl"]
+
+    // HTTP Method
+    let httpMethod = request.httpMethod!
+    switch httpMethod {
+    case "GET": break
+    case "HEAD": components.append("--head")
+    default: components.append("--request \(httpMethod)")
+    }
+
+    // Headers
+    if let headers = request.allHTTPHeaderFields {
+      for (field, value) in headers where field != "Cookie" {
+        let escapedValue = value.replacingOccurrences(of: "\"", with: "\\\"")
+        components.append("--header \"\(field): \(escapedValue)\"")
+      }
+    }
+
+    // Body
+    if let httpBodyData = request.httpBody, let httpBody = String(data: httpBodyData, encoding: .utf8) {
+      var escapedBody = httpBody.replacingOccurrences(of: "\\\"", with: "\\\\\"")
+      escapedBody = escapedBody.replacingOccurrences(of: "\"", with: "\\\"")
+      
+      components.append("--data '\(escapedBody)'")
+    }
+    
+    // Cookies
+    if let cookie = request.allHTTPHeaderFields?["Cookie"] {
+      components.append("--cookie \"\(cookie)\"")
```

Comment:
Thanks for bringing this back! I think `cookie` should be escaped, too, though, just like the other headers above.